sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,166 @@
+  - Fixed bug in hierarchical parallelism implementation related to using a
+    private address of the `parallel_for_work_group` lambda object by all work
+    items in the work group [16f64b8]
+  - Fixed a problem with errors that are reported when `-fsycl-device-only`
+    option is used [7f924a8]
+  - Fixed a crash which happened when a specialization constant is referenced
+    twice [ce020c9]
+  - Fixed a possible filename collision when `-save-temps` is used [99fa86f]
+  - Fixed a hang which could happen on CUDA backend when
+    `sycl::group::async_work_group_copy` is used [f836604]
+
+### SYCL headers and runtime
+  - Ignore exceptions that could happen during sycl objects destruction to avoid
+    terminates [cbd8a72]
+  - Fixed an issue with dependency vector invalidation in some cases [2551b19]
+  - Resolved a conflict with `min`/`max` macro that can be defined by
+    `windows.h` [e7d4537]
+  - Fixed an issue with using wrong CUDA context when creating an event
+    [6788713]
+  - Fixed a crash which happened when no plugin is available [d15de0b]
+  - Fixed float to half-type conversion for small numbers [3a9a1a22]
+  - Fixed `sycl::intel::sub_group::broadcast` which was incorrectly mapped to
+    SPIRV intrinsic [24471bc]
+  - Fixed an issue when a sub-buffer was accessing incorrect memory [4c4054b]
+  - Fixed race which could happen when `sycl::program` API is used from multiple
+    threads [e88a611]
+  - [686e32b]
+  - Make `cl::sycl::event::get` method to be constant [03208c0]
+  - Fixed a crash which could happen when host accessors are created in
+    multiple threads [1d13f84]
+  - Fixed a problem with `sycl::program::set_spec_constant` which set a value
+    for all `sycl::program` objects rather than for an object it was called for
+    only [c22e34b]
+
+### Misc
+  - Fixed an issue with gdb xmethod not working when `run` command is issued
+    multiple times without terminating the session [042d981]
+  - Fixed an issue with platform selection in opencl-aot tool that was caused
+    by an identical supported platform name for CPU and GPU.
+
+## Known issues
+  - The format of the object files produced by the compiler can change between
+    versions. The workaround is to rebuild the application.
+  - The SYCL library doesn't guarantee stable API/ABI, so applications compiled
+    with older version of the SYCL library may not work with new one.
+    The workaround is to rebuild the application.
+    [ABI policy guide](doc/ABIPolicyGuide.md)
+  - Using `cl::sycl::program` API to refer to a kernel defined in another
+    translation unit leads to undefined behavior
+  - Linkage errors with the following message:
+    `error LNK2005: "bool const std::_Is_integral<bool>" (??$_Is_integral@_N@std@@3_NB) already defined`
+    can happen when a SYCL application is built using MS Visual Studio 2019
+    version below 16.3.0
+    The workaround is to enable `-std=c++17` for the failing MSVC version.
+
+## Prerequisites
```

Comment:
I think we should point to exact revision of https://github.com/intel/llvm/blob/sycl/buildbot/dependency.conf used for testing 67d3d9e.

Reply:
I think we should probably just remove prerequisites from release notes, but we need GetStartedGuide.md update first. 
Basically idea is that dependency.conf for corresponding tag or any commit should always point to the recommended dependencies for this commit.
I also don't want to block May'20 release notes by this cleanup needed.

Reply:
#1699 is in progress, so let's remove prerequisites from here.

Reply:
Removed.